vsctl/command/cluster.go

```diff
@@ -140,9 +257,24 @@ func deleteClusterCommand() *cobra.Command {
 		Use:   "delete",
 		Short: "delete a cluster",
 		Run: func(cmd *cobra.Command, args []string) {
-			operator_endpoint, err := cmd.Flags().GetString("operator-endpoint")
+			operator_endpoint, err := getOperatorEndpoint()
+			if err != nil {
+				operator_endpoint, err = cmd.Flags().GetString("operator-endpoint")
+				if err != nil {
+					cmdFailedf(cmd, "get operator endpoint failed: %s", err)
+				}
+			}
+
+			fmt.Print("Deleting a cluster will lose all cluster data, sure you want to delete the vanus cluster(y/n):")
```

Comment:
Deleting a cluster will lose all cluster data and can't be recovered, do you still want to delete the vanus cluster(y/n):